source/ftrack_api/__init__.py

```diff
@@ -21,3 +21,19 @@ def mixin(instance, mixin_class, name=None):
         return
 
     instance.__class__ = type(name, (mixin_class, instance.__class__), {})
+
+
+def support_requests_cert_env():
```

Comment:
It may not be a good idea. this will override the `SSL_CERT_FILE` env support which is the [PEP 476](https://peps.python.org/pep-0476/).
we may want users switch to the new standard env setup?

Reply:
Was under the impression that `load_verify_locations` just added to the existing context meaning that this would be compatible with both?